awscli/customizations/s3/fileinfo.py

```diff
@@ -59,7 +60,8 @@ def save_file(filename, response_data, last_update, is_stream=False):
         with open(filename, 'wb') as out_file:
             write_to_file(out_file, etag, md5, file_chunks)
 
-    if not _is_multipart_etag(etag) and sse != 'aws:kms':
+    if not _is_multipart_etag(etag) and sse != 'aws:kms' and \
```

Comment:
Can we just pull this out into a utils function that takes the entire response data?  Something like can_verify_checksum(response_data).  This is the third time we've needed to update this.  I'm looking to try to localize these changes going forward.  Seems like updating a utils function would be easier.